MediaManager/MediaManagerBase.cs

```diff
@@ -360,6 +360,7 @@ public virtual Task StepForward()
         public event BufferedChangedEventHandler BufferedChanged;
         public event PositionChangedEventHandler PositionChanged;
 
+        public event MediaListFinishedEventHandler MediaListFinished;
```

Comment:
This should not be added i think. There is already https://github.com/Baseflow/XamarinMediaManager/blob/develop/MediaManager/Queue/IMediaQueue.cs#L17